packages/manager/apps/web/client/app/hosting/multisite/delete/hosting-multisite-delete.controller.js

```diff
@@ -46,15 +55,35 @@ angular.module('App').controller(
         wwwNeeded: false,
       };
 
-      this.HostingDomain.getRecords(this.domain, this.subDomain, null)
-        .then(() => {
-          this.selected = { ...this.selected, autoconfigure: true };
-          this.autoConfigureAvailable = true;
-        })
-        .catch(() => {
-          this.selected = { ...this.selected, autoconfigure: false };
-          this.autoConfigureAvailable = false;
-        });
+      this.isOvhDomain = false;
+
+      const getDomainsService = () =>
+        this.Domain.getDomainsService(this.selected.domain.name);
+      const getZoneByZoneName = () =>
+        this.Domain.getZoneByZoneName(this.selected.domain.name);
+      const getAllNameServer = () =>
+        this.Domain.getAllNameServer(this.selected.domain.name);
+
+      promises.push(
+        getDomainsService()
+          .then(() => getZoneByZoneName())
+          .then(() => getAllNameServer())
+          .then(() => {
+            this.isOvhDomain = true;
```

Comment:
**question:** How can you be sure the domain belongs to OVH? Do we have an error if it's not?

Reply:
isOvhDomain is set to false at first. Then, the three API calls needs to return a 200 OK. If so, the domain is an OVH one, so I set the variable to true in the last successful callback. If one of the calls returns an error, then no need to display anything specific I think, the variable isOvhDomain will then stay to false